integration_tests/src/main/python/asserts.py

```diff
@@ -26,23 +26,26 @@
 import difflib
 import sys
 
-def _assert_equal(cpu, gpu, float_check, path):
+def _assert_equal(cpu, gpu, float_check, path, cpu_data_adapted_from = "CPU"):
+    """
+    :param cpu_data_adapted_from: indicate where the cpu result is adapted from, e.g.: pyarrow
```

Comment:
Is this really necessary?  The test should make it obvious which version we're using (either via the test name or the parameters to the test visible in the test name).

Reply:
Done.